src/altinn-app-frontend/src/utils/formComponentUtils.ts

```diff
@@ -535,8 +537,64 @@ export function getFieldName(
   }
 
   if (textResourceBindings.title) {
-    return getTextResourceByKey(textResourceBindings.title, textResources);
+    return smart_unucfirst(
+      getTextResourceByKey(textResourceBindings.title, textResources),
+    );
   }
 
   return getLanguageFromKey('validation.generic_field', language);
 }
+
+/**
+ * Un-uppercase the first letter of a string
+ */
+export function unucfirst(text: string, firstLetterIndex = 0): string {
+  if (firstLetterIndex > 0) {
+    return (
+      text.substring(0, firstLetterIndex) +
+      text[firstLetterIndex].toLowerCase() +
+      text.substring(firstLetterIndex + 1)
+    );
+  }
+  return text[firstLetterIndex].toLowerCase() + text.substring(1);
+}
+
+/**
+ * Un-uppercase the first letter of a string, but be smart about it (avoiding it when the string is an
+ * uppercase abbreviation, etc).
+ */
+export function smart_unucfirst(text: string): string {
```

Comment:
Should perhaps use `smartUnucFirst` as name instead to follow the same naming pattern we have for other functions? Personally I would probably prefre something like `smartLowerCaseFirst` 🙂

Reply:
You're absolutely right! My [years of programming in PHP are shining through](https://news-web.php.net/php.internals/70691).. :laughing: